### PR TITLE
Added Instrumentation Key for chart as a variable

### PR DESCRIPTION
**Pull request by adusumillipraveen**
Added Instrumentation key to chart as a variable .

### JIRA link ###
https://tools.hmcts.net/jira/browse/RPE-879

### Change description ###
Added Instrumentation key to chart as a variable

**Does this PR introduce a breaking change?** (check one with "x")

``` (remove backslash and this comment when copy pasting)
[ ] Yes
[x] No
```

### CODE REVIEW

File: ci-values.yaml [hunk 1]
```diff
@@ -31,3 +31,5 @@ global:
   subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
   tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
   environment: aat
+
+azureApplicationInsightsInstrumentKey: "1a7ce5b7-27e1-4c88-a83b-6e6c358096a7"
```
Comment (timja) [inline ci-values.yaml]: is this a real one? if so probably don't need it

File: java/templates/deployment.yaml [hunk 2]
```diff
@@ -50,8 +50,12 @@ spec:
       containers:
       - image: {{ .Values.image }}
         name: {{ template "hmcts.releaseName" . }}
-        {{- if or .Values.environment .Values.secrets }}
+        {{- if or .Values.environment .Values.secrets .Values.azureApplicationInsightsInstrumentKey }}
```
Comment (timja) [inline java/templates/deployment.yaml]: i wonder if this if statement should go now?
you've added the key as a default so it should always be set?

File: java/values.yaml [hunk 3]
```diff
@@ -16,3 +16,4 @@ livenessDelay: 30
 livenessTimeout: 3
 livenessPeriod: 15
 livenessFailureThreshold: 3
+azureApplicationInsightsInstrumentKey: '00000000-0000-0000-0000-000000000000'
```
Comment (timja) [inline java/values.yaml]: what about
```suggestion
applicationInsightsInstrumentationKey: '00000000-0000-0000-0000-000000000000'
```

Comment (adusumillipraveen) [reply java/values.yaml]: Sure, will change.  keeping it small will be helpful

File: java/templates/deployment.yaml (hunk 2 above)
Comment (adusumillipraveen) [reply java/templates/deployment.yaml]: yes, removed the conditions

**Pull request merged**